docs/profiles.md

```diff
@@ -301,3 +304,35 @@ The following command runs the tests and applies the secrets specified in `profi
     $ inspec exec examples/profile-attribute --attrs examples/profile-attribute.yml
 
 See the full example in the InSpec open source repository: https://github.com/chef/inspec/tree/master/examples/profile-attribute
+
+# Profile files
+
+An InSpec profile may contain additional files that can be accessed during tests. This coveres use-cases where e.g. a list of ports is provided to be tested.
+
+To access these files, they must be stored in the `files` directory at the root of a profile. They are accessed by their name relative to this folder with `inspec.profile.file(...)`.
+
+Here is an example for reading and testing a list of ports. The folder structure is:
```

Comment:
If you decide to change the wording, maybe the example could be a YAML file like so:

```yaml
  web_alpha:
    service_name: httpd-alpha
    port: 80
  web_beta:
    service_name: httpd-beta
    port: 8080
```

... and the test could be:

```ruby
services = YAML.parse(inspec.profile.file('services.yml'))

services.each do |svc|
  describe service(svc['name']) do
    it { should be_running }
  end
  describe port(svc['port']) do
    it { should be_listening }
  end
end
```

Sorry if I'm being pedantic, but I wouldn't want to encourage profiles becoming too abstracted and difficult to follow. I'd want the new `file` functionality to be used when it's really truly necessary.